include/panda_torque_mpc/crocoddyl_reaching.h

```diff
@@ -64,8 +64,8 @@ namespace panda_torque_mpc
             // dummy constructor necessary to use this class as a member variable directly
         }
 
-        CrocoddylReaching(pin::Model _model_pin, const boost::shared_ptr<pin::GeometryModel>& _collision_model ,CrocoddylConfig _config);
-        // CrocoddylReaching(pin::Model _model_pin, CrocoddylConfig _config);
+        CrocoddylReaching(pin::Model model_pin_, const boost::shared_ptr<pin::GeometryModel>& collision_model_ ,CrocoddylConfig _config);
+        // CrocoddylReaching(pin::Model model_pin_, CrocoddylConfig _config);
```

Comment:
```suggestion
        CrocoddylReaching(const pin::Model& model_pin, const boost::shared_ptr<pin::GeometryModel>& collision_model ,const CrocoddylConfig& config);
```
Underscore means internal variable of a class. Parameters of a function should be passed with the naming convention, like with regular variables. The underscore after the variable is used to distinguish it as a class member. The same logic is in python with a leading underscore, being said "protected" variable or function.